src/toolkit/timeseries.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CYCLUS_SRC_TOOLKIT_TIMESERIES_H_
+#define CYCLUS_SRC_TOOLKIT_TIMESERIES_H_
+
+#include <string>
+
+#include "agent.h"
+#include "context.h"
+
+namespace cyclus {
+namespace toolkit {
+
+/// Time series types to be used in the RecordTimeSeries() functions.
+/// These types have the following unit which *must* be adhered to strictly:
+/// - POWER [MWe]
+enum TimeSeriesTypes {
```

Comment:
This should be singular - it makes the it feel more natural when defining/using variables of this type - e.g.

```c++
TimeSeriesTypes mytype; // this is awkward

TimeSeriesType mytype; // more natural
```